### DISCUSSION TITLE
Multiple OneDrive users with their own folders

### DISCUSSION THREAD

**Discussion by roosterveld** [abraunegg/onedrive#2592] [General]
I have 3 family members who's onedrive I'd like to sync with a folder on my server. What is best practice? Create users for each on linux and run onedrive sync under each user separately? Thanks for any feedback!

**Comment by abraunegg** [comment]
@roosterveld 
> Create users for each on linux and run onedrive sync under each user separately?

Correct - that is the best way to do this - additionally then if they login to the system, they have their running instance, with the files & folders correctly attributed and permissioned to their user account, with their data stored in the correct location `~/OneDrive` or stored centrally elsewhere, but then symbolically linked to `~/OneDrive`

The alternative is to set them up using a central Linux account, but then using the `--confdir` option to specify unique 'sync_dir' paths for each user, and having that data then exposed for the users to access. You will need to take care on what user|group the client is running as, what permissions are being given to the data (usually increasing to either include group permissions or world permissions) so that the users can access the data as required.

**Reply by roosterveld** [reply]
Thank you!!
